subprojects/launcher/build.gradle.kts

```diff
@@ -17,6 +17,7 @@ dependencies {
     implementation(project(":files"))
     implementation(project(":file-collections"))
     implementation(project(":snapshots"))
+    implementation(project(":execution"))
```

Comment:
Why is this needed now?

Reply:
The definition of `WorkInputListener(s)` is in execution, and we are using that for continuous build.